.github/workflows/tests-unit-php.yml

```diff
@@ -2,10 +2,26 @@ name: PHP Unit Tests
 
 on:
   push:
+    # Only run if PHP-related files changed.
+    paths:
+      - '**.php'
+      - 'phpunit.xml.dist'
+      - 'phpunit-multisite.xml.dist'
+      - 'composer.json'
+      - 'composer.lock'
+      - 'tests/phpunit/**'
```

Comment:
```suggestion
      - 'tests/phpunit/**',
      - 'includes/**'
```

Reply:
No commas.

`includes/` should be covered by `**.php` at the top.

Reply:
What about https://github.com/google/web-stories-wp/blob/main/includes/data/stories/demo.json ?

Reply:
You're right. It's used by the `test_json_file_contains_replaceable_urls` test